src/Microsoft.Extensions.Localization/Internal/AssemblyResourceStringProvider.cs

```diff
@@ -65,6 +63,7 @@ private IList<string> ThrowOrNull(CultureInfo culture, bool throwOnMissing)
         public IList<string> GetAllResourceStrings(CultureInfo culture, bool throwOnMissing)
         {
             var cacheKey = GetResourceCacheKey(culture);
+
```

Comment:
Yes, please get rid of the extra noise. Style consistency is not something we're shooting for, especially in files you wouldn't otherwise touch.